mesonbuild/dependencies/pkgconfig.py

```diff
@@ -6,7 +6,16 @@
 from pathlib import Path
 
 from .base import ExternalDependency, DependencyException, sort_libpaths, DependencyTypeName
-from ..mesonlib import EnvironmentVariables, OrderedSet, PerMachine, Popen_safe, Popen_safe_logged, MachineChoice, join_args
+from ..mesonlib import (
+    EnvironmentVariables,
+    OrderedSet,
+    PerMachine,
+    Popen_safe,
+    Popen_safe_logged,
+    MachineChoice,
+    join_args,
+    MesonException,
+)
```

Comment:
Please use a less noisy import style: https://pycqa.github.io/isort/docs/configuration/multi_line_output_modes.html#5-hanging-grid-grouped

(Yes, I know this isn't enforced. Getting automatic code formatters to actually do what you want is extremely challenging. In fact I have an open bug report on the isort issue tracker that is probably never going to be fixed, which means no isort for me! Still, style number 3 -- a.k.a. "black deformatter" style -- is IMO dreadful.)

Reply:
I have Ruff configured to autoformat things and as I was creating the patch I knew these changes were not right, but I was hoping nobody would notice also knowing that someone would end up noticing :)

I will fix all of these.